dapr/proto/runtime/v1/dapr.proto

```diff
@@ -511,8 +514,25 @@ message SubscribeConfigurationRequest {
   map<string, string> metadata = 3;
 }
 
+// UnSubscribeConfigurationRequest is the message to stop watching the key-value configuration.
+message UnsubscribeConfigurationRequest {
+  // The name of configuration store.
+  string store_name = 1;
+
+  // The id to unsubscribe.
+  string id = 2;
+}
+
 message SubscribeConfigurationResponse {
+  // Subscribe id, used to stop subscription.
+  string id = 1;
+
   // The list of items containing configuration values
-  repeated common.v1.ConfigurationItem items = 1;
+  repeated common.v1.ConfigurationItem items = 2;
```

Comment:
@artursouza @yaron2 Please not this change will break existing implementations of the configuration API particularly the subscribe call.

Reply:
It's an Alpha API